### ISSUE TITLE
dementia

### ISSUE THREAD

**Issue by mariaobon** [obophenotype/human-phenotype-ontology#10507]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This term already exists, see Dementia HP:0000726
